dae/dae/backends/storage/impala_genotype_storage.py

```diff
@@ -24,11 +24,10 @@
 
 
 class ImpalaGenotypeStorage(GenotypeStorage):
-    """Storage that uses impala as its backend."""
+    """Defines Apache Impala genotype storage."""
```

Comment:
I think the comment should be in imperative case i.e. "Define" instead of "Defines".